webpack.config.js

```diff
@@ -1,20 +1,31 @@
 const path = require('path');
 
-module.exports = {
+module.exports = (env, argv) => ({
     entry: './src/Synergism.ts',
-    mode: 'none',
+    mode: argv.mode,
+    devtool: argv.mode === 'development' ? 'eval-source-map' : 'source-map',
     module: {
         rules: [{
             test: /\.tsx?$/,
             use: 'ts-loader',
             exclude: /node_modules/,
         }],
+        
     },
     resolve: {
-        extensions: [ '.ts' ],
+        extensions: [ '.ts', '.js' ],
```

Comment:
no js files should be resolved

Reply:
webpack dev server needs thatm otherwise it won't run.